examples/example-expo-latest/package.json

```diff
@@ -21,7 +21,7 @@
     "expo-localization": "~15.0.3",
     "expo-status-bar": "~1.12.1",
     "posthog-react-native": "file:.yalc/posthog-react-native",
-    "posthog-react-native-session-replay": "^1.1.0",
```

Comment:
have to release https://github.com/PostHog/posthog-react-native-session-replay/pull/30 first